### ISSUE TITLE
Understanding 1.4.3 and 1.4.6 should refer to 1.4.11

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#1261]
Understanding 1.4.3 and 1.4.6 both contain the sentence:

> Although this Success Criterion only applies to text, similar issues occur for content presented in charts, graphs, diagrams, and other non-text-based information. Content presented in this manner should also have good contrast to ensure that more users can access the information. 

This sentence is outdated and should be removed or adapted. Instead, reference should be made to SC 1.4.11
